board.go

```diff
@@ -0,0 +1,69 @@
+package honeycombio
+
+// Compile-time proof of interface implementation.
+var _ Boards = (*boards)(nil)
+
+// Boards describes all the boards related methods that Honeycomb supports.
```

Comment:
Actually this is a lie, it doesn't support delete, but neither does the marker struct I copied from

Reply:
disregard, 144892cad94277b7f9e56a59dd55d1f9bb7f8802